src/main/java/org/broadinstitute/hellbender/tools/copynumber/denoising/HDF5SVDReadCountPanelOfNormals.java

```diff
@@ -154,7 +155,15 @@ public double getVersion() {
 
     @Override
     public int getNumEigensamples() {
-        return (int) file.readDouble(PANEL_NUM_EIGENSAMPLES_PATH);
+        try {
+            return (int) file.readDouble(PANEL_NUM_EIGENSAMPLES_PATH);
+        } catch (final HDF5LibException e) {
```

Comment:
Could you use a less general exception to catch this?

Reply:
This is the exception thrown by both our wrapper and the Java HDF5 library, unfortunately---there's no more specific exception.